backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -315,3 +318,10 @@ function isCreationValid(creations: number[], amount: number, creationDate: Date
   }
   return true
 }
+async function hasActivatedEmail(email: string): Promise<boolean> {
+  const repository = getCustomRepository(LoginUserRepository)
+  const user = await repository.findByEmail(email)
+  let emailActivate = false
+  if (user) emailActivate = user.emailChecked
+  return emailActivate
```

Comment:
```suggestion
  return user ? user.emailChecked : false
```
And no variable needed